src/dataDisplay/image/image.jsx

```diff
@@ -96,7 +96,7 @@ function Image(props) {
                     compact
                     size={avatarSize}
                     title="This person has no image"
-                    type="user"
+                    type={type}
```

Comment:
I am thinking we should change the `defaultProp.type` to 'user'.

Reply:
Yeah, that might help minimize any possibility of regressions.  `undefined` as the default feels more correct to me in the general sense: i.e. not all Images are Avatar Images (in fact, most probably aren't), but, due to the preexisting bug where this was hard-coded to "user", making that default might be safer.

Reply:
No, it shouldn't have a default prop value because it'd override always the `src` prop in line 88.

```
<ElementType
        className={containerClasses}
        src={!type ? src : null}                        // <--------- here
        style={newStyle}
>
```

Reply:
... and I suppose the we're in a condition here where `type` is always going to be defined. But I think the original thinking here is that the icon should always be the user icon (image below). Is that a different case now?

![image](https://user-images.githubusercontent.com/1100790/140553443-4c9230b4-85df-402b-aa4a-f9c30c16f4b8.png)

Reply:
I checked the whole code and I think it works with a default prop so I defined it, thanks.

Reply:
Okay, i am good with this then. Thank you for clarifying.